wp-content/themes/missilethreat/template-parts/sidebar.php

```diff
@@ -0,0 +1,47 @@
+<?php
+/**
+ * Sidebar for archive pages
+ *
+ * @package CSIS iLab
+ * @subpackage @package MissileThreat
+ * @since 1.0.0
+ */
+
+function get_archive_post_type() {
+	return is_archive() ? get_queried_object()->name : false;
+}
+
+if( get_archive_post_type() ) {
```

Comment:
```suggestion
if( is_post_type_archive( array('defsys', 'missile')  ) {
```

You want to check if you're on a post type archive, which is different than returning the post archive name (which is what `get_archive_post_type()` does.